src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -4,6 +4,12 @@ public abstract class Machine implements Workable {
     private String name;
     private String color;
 
+    public Machine() {
+    }
+
+    public Machine(String bull1, String red) {
```

Comment:
bull1? what is that, how it is related to machine, why do you create constructor which do not initialize fileds?